lib/config.py

```diff
@@ -50,6 +50,12 @@
         dict(help='Keep build directory and its logs and artifacts.',
              action='store_true'),
 }
+DISTRO_ARGS = {
+    ('--distro-name',):
+        dict(help='Base Linux distribution', default=''),
```

Comment:
Why have this default? The default value `None` is just as good, and it will be replaced by the value in `config.yaml` anyway.